pkg/component/observability/plutono/plutono.go

```diff
@@ -425,6 +425,9 @@ func (p *plutono) getDashboardConfigMap(ctx context.Context, suffix string) (*co
 			if !p.values.VPNHighAvailabilityEnabled {
 				ignorePaths.Insert("ha-vpn")
 			}
+			if p.values.VPNHighAvailabilityEnabled {
+				ignorePaths.Insert("envoy-proxy")
+			}
```

Comment:
Why not simply use an `else` block if the inverse condition is already checked above?